index.js

```diff
@@ -1,14 +1,39 @@
 /***** Deliverable 1 *****/
+// console.log("Here's your header:", header)
+
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+
+function headerColorChange (){
+    if (header.style.color === "red"){
+        header.style.color = "black"
+    }
+    else {
+        header.style.color = "red"
+    }
+}
+
+header.addEventListener("click", headerColorChange);
```

Comment:
💯 great use of passing this helper function that we already had to your eventListener. I'm guessing you overwriting it has something to do with your second comment?